aws/resource_aws_lb_test.go

```diff
@@ -874,6 +919,75 @@ resource "aws_security_group" "alb_test" {
 }`, lbName)
 }
 
+func testAccAWSLBConfig_basicWithHttp2Toggle(lbName string, http2 bool) string {
```

Comment:
Minor nitpick: generally we would just name this `testAccAWSLBConfig_enableHttp2`